package.json

```diff
@@ -92,11 +92,8 @@
   "repository": "ForestAdmin/toolbelt",
   "scripts": {
     "lint": "./node_modules/eslint/bin/eslint.js src test",
-    "postpack": "rm -f oclif.manifest.json",
-    "prepack": "oclif-dev manifest && oclif-dev readme",
```

Comment:
`oclif-dev` is not part of the dev dependencies, thus the error below:
https://travis-ci.com/github/ForestAdmin/toolbelt/jobs/336155697#L308

It has been setup (by default I guess), 2 years ago, but that is not necessary in my opinion.
So let's simplify all this.